### ISSUE TITLE
Image size inconsistency for visualize.visualization() 

### ISSUE THREAD

**Issue by LoveHRTF** [LoveHRTF/YouOnlyLookOnce-TF2.0#3]
logits = np.squeeze(model(img))

img should be [448, 448, 3]
Currently random size
